src/Components/ArticlesList.jsx

```diff
@@ -0,0 +1,56 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import ArticleCard from "./ArticleCard.jsx";
+
+function ArticlesList() {
+  const [allArticles, setAllArticles] = useState([]);
+  const [error, setError] = useState(false);
+
```

Comment:
Please add a state for loading and display a message to the user while the articles are being loaded.